.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Refractr
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Publish
+        run: doit publish
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: Show details
```

Comment:
The 'doit show' wasn't included in the original publish command in travis-ci. Is it intentional that it's included here, and is it okay that it is _after_ the publish command?

Reply:
The `show` part was included in `publish` command, and as `test` and `publish` are running separately in this case, I put the show away from `publish` so it can be used after running test or publish the same way.
I also removed the decision if it will be publish or not from `refractr/cfg.py` and let separated files, test and publish, to be more clear.